package com.exception;

public class ExceptionTest {
	public static void main(String[] args) {
		int result = 0;
		int a =13, b=0;
		
		try {
			result = a/b;
			System.out.println(result);
		}catch(ArithmeticException e) {
			result = 9999;
		}
		
	}
}
